src/containers/CreateAccount/CreateAccount.scss

```diff
@@ -0,0 +1,43 @@
+@use "../../assets/sass/variables" as *;
+
+@media only screen and (max-width: $max-mobile-view) {
```

Comment:
I don't think this needs to be in a media query? The styles can just be applied whatever.